pom.xml

```diff
@@ -16,7 +16,10 @@
   <properties>
     <jenkins.version>1.609</jenkins.version>
     <java.level>6</java.level>
-    <workflow.version>1.4.3</workflow.version>
+    <workflow-job.version>1.4.3</workflow-job.version>
+    <workflow-cps.version>1.4.3</workflow-cps.version>
```

Comment:
@olivergondza I've bumped the Jenkins core version and other dependencies to minimum required versions to fully test the symbol annotation